### PR TITLE
Improve repo setup

### PR DESCRIPTION
**Pull request by 0x009922**
**Note:** look it after merge of https://github.com/soramitsu/soramitsu-js-ui-library/pull/328

---

- [x] `ui` types fix: separate Storybook & Ko types from global scope of main code -> Volar works
- [x] `ui` build feature: actual `*.d.ts` declarations for components, thus actual component types in the final bundle
- [x] `ui` types fix: separate Cypress tests from global scope of main code -> possibility for future to use jest tests without type conflicts
- [ ] Update `theme` package rollup config
- [x] `ui` build feature: sourceMaps by vite?

### CODE REVIEW

**Review by alexnatalia — APPROVED** [review]

**Review by stefashkaa — APPROVED** [review]

File: packages/ui/.storybook/main.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ function resolve(...paths) {
 }
 
 module.exports = {
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```
Comment (stefashkaa) [inline packages/ui/.storybook/main.js:9]: why did you remove `mdx`? I guess we'll need it

Comment (0x009922) [reply packages/ui/.storybook/main.js:9]: Because I don't know yet where `mdx` will be sourced, in which directory.
